.github/workflows/ci.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.18.x,1.19.x]
```

Comment:
Also, if you don't mind - please file an issue to update linting to Go 1.20. (I assume it was enough work that you didn't want to do it in this PR.)